src/firestore/firekit.ts

```diff
@@ -1743,6 +1751,42 @@ export class RoarFirekit {
     return task;
   }
 
+  async updateTaskOrVariant(updateData: UpdateTaskVariantData) {
+    try {
+      this._verifyAuthentication();
+    } catch (error) {
+      console.log('auth error: ', error);
+    }
+
+    try {
+      this._verifyAdmin();
+    } catch (error) {
+      console.log('admin error: ', error);
+    }
+
+    let docRef;
+    let data;
```

Comment:
And if you accept the interface suggestion above, then you could use.
```suggestion
    const { data } = updateData;
```